crates/oci/src/client.rs

```diff
@@ -246,16 +274,43 @@ impl Client {
         &mut self,
         locked: &mut LockedApp,
         assembly_mode: AssemblyMode,
+        compose_mode: ComposeMode,
     ) -> Result<Vec<ImageLayer>> {
-        let mut layers = Vec::new();
```

Comment:
@itowlson Calling attention to this diff because it's minimized by default. Let me know if this is easier to follow.